.github/workflows/release.yml

```diff
@@ -106,3 +106,19 @@ jobs:
           tag-name: ${{ env.TAG }}
         env:
           COMMITTER_TOKEN: ${{ secrets.COMMITTER_TOKEN }}
+
+  winget-update:
+    runs-on: ubuntu-latest
```

Comment:
I know komac has linux and macos binaries, but does this workflow work on linux? The readme uses `windows-latest` in its examples.

Reply:
Hmm I see I guess it works then. I think linux start up faster on actions anyways so it's probably better.